src/dymaptic.GeoBlazor.Core/Components/Layers/FeatureLayer.cs

```diff
@@ -294,13 +295,18 @@ public FeatureLayer()
     /// <exception cref="InvalidOperationException">
     ///     If the layer is already loaded, you must use <see cref="ApplyEdits"/> to add graphics.
     /// </exception>
-    public Task Add(Graphic graphic)
+    public async Task Add(Graphic graphic)
     {
         if (JsLayerReference is not null)
         {
-            throw new InvalidOperationException("Cannot add graphics to a feature layer that is already loaded. Use ApplyEdits instead.");
+            await ApplyEdits(new FeatureEdits
```

Comment:
@seahro here you can see where I removed the exception and re-implemented using `ApplyEdits`